src/base/bittorrent/private/bandwidthscheduler.cpp

```diff
@@ -93,7 +93,7 @@ bool BandwidthScheduler::isTimeForAlternative() const
 
 void BandwidthScheduler::onTimeout()
 {
-    bool alternative = isTimeForAlternative();
+    const bool alternative = isTimeForAlternative();
```

Comment:
I think const should be added after reviewing the usage of the variable (not sure if you are doing it already), adding it everywhere without context might not be good for readability in some cases.

Reply:
That's what i was afraid of.. but on the other hand you take a look at this and you know this var is not changed later on.